aspnet/fundamentals/localization/sample/Controllers/AccountController.cs

```diff
@@ -0,0 +1,467 @@
+﻿using System;
```

Comment:
If the code is being referenced then it can stay. We do in general want to minimize the amount of extra code that we have to maintain in these samples. Extra code can also distract from the key features that the sample is trying to demonstrate. There's no need to have a full starter web app when you just want to show one feature. Since the membership code in this case is localized and this article is about localization then I think it's ok to keep it.